ui/src/shared/components/ConfirmButtons.js

```diff
@@ -17,6 +28,10 @@ ConfirmButtons.propTypes = {
   onConfirm: func.isRequired,
   item: oneOfType([shape(), string]),
   onCancel: func.isRequired,
+  buttonSize: string,
 }
 
+ConfirmButtons.defaultProps = {
+  buttonSize: 'btn-sm',
```

Comment:
Oh, but looks like it's always present, so don't need to worry about `null`. 👍 Good practice